addons/mail/static/src/model/model_field.js

```diff
@@ -606,12 +651,23 @@ class ModelField {
             }
             isDeleting = true;
```

Comment:
`isUnlinking`

Reply:
For this one I'll use `hasChanged` to stay consistent with other such variables.